src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/Methodologies/MethodologyRepositoryTests.cs

```diff
@@ -0,0 +1,310 @@
+                Release = release2,
+                Role = ReleaseRole.Contributor,
+            };
+            var methodology2 = new Methodology
+            {
+                Id = Guid.NewGuid(),
+                Title = "Test methodology 2",
+                Publications = new List<Publication> {publication2}
+            };
+            
+            var publication3 = new Publication
+            {
+                Id = Guid.NewGuid(),
+                Title = "Test pub 3"
+            };
+            var release3 = new Release
+            {
+                Id = Guid.NewGuid(),
+                PublicationId = publication3.Id
+            };
+            var userReleaseRole3 = new UserReleaseRole
+            {
+                UserId = Guid.NewGuid(),
+                Release = release3,
+                Role = ReleaseRole.Lead,
+            };
+            var methodology3 = new Methodology
+            {
+                Id = Guid.NewGuid(),
+                Title = "Ignored methodology 3",
+                Publications = new List<Publication> {publication3}
+            };
+            
+            var methodology4 = new Methodology
+            {
+                Id = Guid.NewGuid(),
+                Title = "Ignored methodology 3",
+                Publications = new List<Publication>()
+            };
+            
+            var contentDbContextId = Guid.NewGuid().ToString();
+            await using (var contentDbContext = DbUtils.InMemoryApplicationDbContext(contentDbContextId))
+            {
+                await contentDbContext.AddAsync(publication1);
+                await contentDbContext.AddAsync(release1);
+                await contentDbContext.AddAsync(userReleaseRole1);
+                await contentDbContext.AddAsync(methodology1);
+                
+                await contentDbContext.AddAsync(publication2);
+                await contentDbContext.AddAsync(release2);
+                await contentDbContext.AddAsync(userReleaseRole2);
+                await contentDbContext.AddAsync(methodology2);
+                
+                await contentDbContext.AddAsync(publication3);
+                await contentDbContext.AddAsync(release3);
+                await contentDbContext.AddAsync(userReleaseRole3);
+                await contentDbContext.AddAsync(methodology3);
+                
+                await contentDbContext.AddAsync(methodology4);
+                
+                await contentDbContext.SaveChangesAsync();
+            }
+
+            await using (var contentDbContext = DbUtils.InMemoryApplicationDbContext(contentDbContextId))
+            {
+                var methodologyRepository = new MethodologyRepository(contentDbContext);
+                var methodologies = await methodologyRepository.GetMethodologiesForUser(userId);
+
+                Assert.Equal(2, methodologies.Count);
+                Assert.Contains(methodologies, m => m.Id == methodology1.Id);
```

Comment:
Oh nice! Didn't realize this existed, definitely gonna use it myself from now on :+1: